### ISSUE TITLE
Upgrading to Django 2.1.7, Python 3.6 and 0.13b2: urlmatching not working

### ISSUE THREAD

**Issue by applantation** [deschler/django-modeltranslation#492]
I am in the process of upgrading a working installation to Django 2.1.7 on Python 3.6. I am using 0.13b2.

But when I now run the app, I get the following 404 when I call up the homepage.

```
Using the URLconf defined in urls, Django tried these URL patterns, in this order:
en/
...
The empty path didn't match any of these.
```

So, it inserts the default language correctly, but then cannot find the urlpath. This was working before. There are no further error messages (also nothing in debug mode)

As I said, this was working before the upgrade.

**Comment by last-partizan** [comment]
Can you show urlconf and related models and translation.py?

**Comment by applantation** [comment]
This turned out to be a hidden exception behind the 404 (never eat exceptions), that was ultimately caused by not correctly upgrading to version 0.13b. This triggered the '_clone() got an unexpected keyword argument '_rewrite'' exception that I did not see.

The old modeltranslation version 0.12.2 was not correctly uninstalled and was then picked up instead of the newer one. I finally found it by printing out the modeltranslation version number. Doh.
